doc/troubleshooting/precision.md

```diff
@@ -0,0 +1,73 @@
+# Why does a model have low precision?
+
+Many phenomena are caused by model accuracy.
+For example, during simulations, temperatures explode, structures fall apart, and atoms are lost.
+One can [test the model](../test/test.md) to confirm whether the model has the enough accuracy.
+
+There are many reasons for a low-quality model.
+Some common reasons are listed below.
+
+## Data
+
+### Data units and signs
+
+The unit of training data should follow what is listed in [data section](../data/system.md).
+Usually, the package to calculate the training data has different units from those of the DeePMD-kit.
+It is noted that some software label the energy gradient as forces, instead of the negative energy gradient.
+It is neccessary to check them carefully to avoid inconsistent data.
+
+### SCF coverage and data accuracy
```

Comment:
We may also want to discuss the convergence of dft data with respect to energy cutoff and k-space sampling.

Reply:
Sorry I am not familiar with the plane wave. Maybe you can revise it by yourself...